cli/command/base.go

```diff
@@ -65,18 +68,41 @@ func (b *CommandBase) deleteHelper(c *cli.Context, entityType string, deleteFN f
 		return err
 	}
 
-	if c.GlobalBool(config.FLAG_NO_WAIT) {
-		// todo: use single 'running as job' helper printer
-		b.printer.Printf("Running as job '%s'", jobID)
+	if waitFlag {
+		b.printJobHelper(jobID)
 		return nil
 	}
 
 	b.printer.StartSpinner("deleting")
 	defer b.printer.StopSpinner()
 
-	if _, err := client.WaitForJob(b.client, jobID, c.GlobalDuration(config.FLAG_TIMEOUT)); err != nil {
+	if _, err := client.WaitForJob(b.client, jobID, waitTimeout); err != nil {
 		return err
 	}
 
 	return nil
 }
+
+func (b *CommandBase) waitOnJobHelper(c *cli.Context, jobID string, onCompleteFN func(entityID string) error) error {
+	waitFlag := c.GlobalBool(config.FLAG_NO_WAIT)
+	waitTimeout := c.GlobalDuration(config.FLAG_TIMEOUT)
+
+	if waitFlag {
+		b.printJobHelper(jobID)
+		return nil
+	}
+
+	b.printer.StartSpinner("creating")
```

Comment:
I realized I made a mistake - this is probably going to be used for `updates` as well, which means the spinner text will change.